Add blog post about binary artifact signing (#37639)
kubernetes/website: content/en/blog/_posts/2022-12-12-kubernetes-release-artifact-signing.md

`kubectl.sig` and `HashedRekordObj.signature.publicKey.content` should be
identical with the contents of `kubectl.cert`. It is also possible to specify
the remote certificate and signature locations without downloading them:

```shell
COSIGN_EXPERIMENTAL=1 cosign verify-blob kubectl \
    --signature https://dl.k8s.io/release/v1.26.0-rc.1/bin/linux/amd64/kubectl.sig \
    --certificate https://dl.k8s.io/release/v1.26.0-rc.1/bin/linux/amd64/kubectl.cert
```

```
tlog entry verified with uuid: 5d54b39222e3fa9a21bcb0badd8aac939b4b0d1d9085b37f1f10b18a8cd24657 index: 8173886
Verified OK
```

All of the mentioned steps as well as how to verify container images are
outlined in the official documentation about how to [Verify Signed Kubernetes
Artifacts][docs]. In one of the next upcoming Kubernetes releases we will
working making the global story more mature by ensuring that truly all
Kubernetes artifacts are signed. Beside that, we consider to use Kubernetes
owned infrastructure for the signing (root trust) and verification (transparency
log) process.

[docs]: /docs/tasks/administer-cluster/verify-signed-artifacts

> 

Reviewer: (optional)
```suggestion

## Getting involved

```

If you're interested in contributing to SIG Release, then consider applying for
the upcoming v1.27 shadowing program (watch for the announcement on
[k-dev][k-dev]) or join our [weekly meeting][meeting] to say _hi_.

> We're looking forward to shipping more of those awesome features in future

Reviewer: ```suggestion
We're looking forward to making even more of those awesome changes for future
```
because the branch rename isn't really a feature.
